Godeps/Godeps.json

```diff
@@ -18,6 +18,11 @@
 			"Rev": "'75cd24fc2f2c2a2088577d12123ddee5f54e0675'"
 		},
 		{
+			"ImportPath": "github.com/BurntSushi/toml",
```

Comment:
For a start we CAN NOT use this dep - it is licensed under the WTFPL which is not allowed.

https://news.ycombinator.com/item?id=5733050#up_5733477